code/obj/item/pinpointer.dm

```diff
@@ -203,6 +203,13 @@
 		STOP_TRACKING_CAT(TR_CAT_NUKE_OP_STYLE)
 		..()
 
+/obj/item/pinpointer/identificationcomputer
+	name = "pinpointer (identification computer)"
+	desc = "Points in the direction of the portable identification computer."
+	icon_state = "id_pinoff"
+	icon_type = "id"
+	target_criteria = /obj/machinery/computer/card/portable
```

Comment:
there are other this object in the world, there's no guarantee which one this would point to the way pinpointers are currently coded

Reply:
In my testing it worked fine because the portable version is one of a kind. But given you have more experience with this sort of thing, I would like to know what you would recommend be done.

Reply:
I was mistaken this probably the only one

There are a few edge cases in which this will still break but it's an issue for every pinpointer.